### ISSUE TITLE
ImportError: cannot import name 'SessionInfo' from 'streamlit.runtime.runtime'

### ISSUE THREAD

**Issue by isezen** [whitphx/streamlit-server-state#171]
Streamlit version is `1.18.1`.

ERROR:
```
ImportError: cannot import name 'SessionInfo' from 'streamlit.runtime.runtime' (/home/ziesha/.local/lib/python3.10/site-packages/streamlit/runtime/runtime.py)
```

```
File "/home/ziesha/.local/lib/python3.10/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 565, in _run_script
    exec(code, module.__dict__)
File "/home/ziesha/ziesha-pool/Home.py", line 23, in <module>
    from Faucet import send_zsh
File "/home/ziesha/ziesha-pool/Faucet.py", line 11, in <module>
    from streamlit_server_state import server_state as _ss
File "/home/ziesha/.local/lib/python3.10/site-packages/streamlit_server_state/__init__.py", line 3, in <module>
    from .rerun import make_force_rerun_bound_sessions
File "/home/ziesha/.local/lib/python3.10/site-packages/streamlit_server_state/rerun.py", line 1, in <module>
    from .rerun_suppression import is_rerun_suppressed
File "/home/ziesha/.local/lib/python3.10/site-packages/streamlit_server_state/rerun_suppression.py", line 3, in <module>
    from .app_context import get_app_context
File "/home/ziesha/.local/lib/python3.10/site-packages/streamlit_server_state/app_context.py", line 3, in <module>
    from ._compat import (
File "/home/ziesha/.local/lib/python3.10/site-packages/streamlit_server_state/_compat.py", line 56, in <module>
    from streamlit.runtime.runtime import SessionInfo
```

**Comment by whitphx** [comment]
Thank you for the report.
The fix will be released in v0.15.2. Please try it.

**Comment by whitphx** [comment]
Resolved with  #172, #175
